bin/docker/supervisor.conf

```diff
@@ -11,9 +11,14 @@ stdout_logfile=logs/docker.log
 stdout_logfile_maxbytes=1MB
 stopsignal=KILL
 environment=
+    SOLITUDE_BRAINTREE_PROXY="http://solitude-auth:2603/v1/braintree/",
```

Comment:
I think it would be just as fine to call this ``SOLITUDE_BRAINTREE_AUTH`` - might as well fully commit to "auth" 100% ;)